Cargo.toml

```diff
@@ -66,15 +67,15 @@ test = ["test-blas-openblas-sys"]
 # This feature is used for docs
 docs = ["approx", "serde", "rayon"]
 
-std = ["num-traits/std"]
+std = ["num-traits/std", "matrixmultiply/std"]
 
 [profile.release]
 [profile.bench]
 debug = true
 
 [workspace]
 members = ["ndarray-rand", "serialization-tests", "blas-tests"]
-exclude = ["numeric-tests"]
+exclude = ["numeric-tests","ensure_no_std"]
```

Comment:
What's this `ensure_no_std`, it sounds like a good idea but I don't think it exists in this PR

Reply:
I tried to refer to the no_std of matrixmultiply, But because I don’t know much about CI,  this idea cannot be put into practice at currect :(